### PR TITLE
Populate calibration point type 1 in DB

### PR DESCRIPTION
**Pull request by peltsi212**
VIITE-2351

### CODE REVIEW

File: digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+UPDATE CALIBRATION_POINT cpu SET "TYPE" = (SELECT CASE
+-- leave expired calibration points to original value 2
+		WHEN (cp.VALID_TO IS NOT NULL) THEN 2
```
Comment (andre-policarpo) [inline digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql]: I would remove this and just let the expired ones with the type they have - doesn't matter which is.
And add an `where` clause to the end.

Comment (peltsi212) [reply digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql]: fixed

File: digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
+UPDATE CALIBRATION_POINT cpu SET "TYPE" = (SELECT CASE
+-- leave expired calibration points and user assigned points to original value
+-- [TYPE = 2] Includes templates, points where ADDR_M is equal to START_ADDR_M or END_ADDR_M of the road (road_number, road_part_number and track) and when ROAD_TYPE changes
+		WHEN (rp.ADDR_M = (SELECT MAX(roadAddr.END_ADDR_M) FROM ROADWAY roadAddr
```
Comment (skosonen) [inline digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql]: Why there is this MAX(roadAddr.END_ADDR_M)? I think all discontinuity places should be included.

Comment (peltsi212) [reply digiroad2-oracle/src/main/resources/db/migration/V0_32__calibration_point_type_update.sql]: fixed

**Pull request merged**